src/main/java/dev/meyi/bn/handlers/ChestTickHandler.java

```diff
@@ -73,31 +40,33 @@ public static void updateBazaarOrders(IInventory chest) {
         }
 
         String displayName = StringUtils
-            .stripControlCodes(chest.getStackInSlot(i).getDisplayName().split(": ")[1]);
+                .stripControlCodes(chest.getStackInSlot(i).getDisplayName().split(": ")[1]);
         String type = StringUtils.stripControlCodes(
-            chest.getStackInSlot(i).getDisplayName().split(": ")[0].toLowerCase());
+                chest.getStackInSlot(i).getDisplayName().split(": ")[0].toLowerCase());
 
         if (BazaarNotifier.bazaarConversionsReversed.has(displayName)) {
           int amountLeft = -1;
           double price;
+          String priceString;
           if (lore.get(4).toLowerCase().contains("expire")) {
-            price = Double.parseDouble(
-                StringUtils.stripControlCodes(lore.get(6)).replaceAll(",", "").split(" ")[3]);
+            priceString = StringUtils.stripControlCodes(lore.get(6)).replaceAll(",", "")
+                    .split(" ")[3];
+            price = Double.parseDouble(priceString);
           } else if (lore.get(5).toLowerCase().contains("expire")) {
-            price = Double.parseDouble(
-                StringUtils.stripControlCodes(lore.get(7)).replaceAll(",", "").split(" ")[3]);
+            priceString = StringUtils.stripControlCodes(lore.get(7)).replaceAll(",", "")
+                    .split(" ")[3];
+            price = Double.parseDouble(priceString);
           } else {
-            price = Double
-                .parseDouble(
-                    StringUtils
-                        .stripControlCodes(lore.get((lore.get(3).startsWith("Filled:")) ? 5 : 4)
-                            .replaceAll(",", "").split(" ")[3]));
+            priceString = StringUtils.stripControlCodes(
+                    lore.get((lore.get(3).startsWith("Filled:")) ? 5 : 4).replaceAll(",", "")
+                            .split(" ")[3]);
+            price = Double.parseDouble(priceString);
           }
           int orderInQuestion = -1;
           for (int j = 0; j < BazaarNotifier.orders.length(); j++) {
             JSONObject order = BazaarNotifier.orders.getJSONObject(j);
-            if (Double.compare(order.getDouble("pricePerUnit"), price) == 0 && type
-                .equals(order.getString("type"))) {
+            if (priceString.equalsIgnoreCase(order.getString("priceString")) && type
+                    .equals(order.getString("type"))) {

```

Comment:
Any reason for the changes to this file?

Reply:
You did the chances to the master and I copied your changed files